internal/kms/pubsub.go

```diff
@@ -194,17 +198,23 @@ func (s *pubSubService) requestEncryptionKeyFromPeers(
 	return nil
 }
 
+type fetchEncryptionKeyReply struct {
+	Links              [][]byte
+	Blocks             [][]byte
+	EphemeralPublicKey []byte
+}
```

Comment:
nitpick/question: Why `fetchEncryptionKeyReply` and not `fetchEncryptionKeyResponse`?

Reply:
This is just copy pasted from what we had before. It's also how google names the response parameter (reply).